frontend/packages/ux-editor/src/components/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
This page is temporary until we get a new design from UX. The designers have approved this page until the new design is ready. :)